.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,77 @@
+# docs https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+
+name: Create a bug report
+description: Errors and regression reports with complete reproducing test cases and/or stack traces.
+labels: [requires triage]
+body:
+  - type: markdown
+    attributes:
+      value: "
+**GUIDELINES FOR REPORTING BUGS**
```

Comment:
I'm trying this out at testgerrit:   https://github.com/sqlalchemy/testgerrit/issues/new?assignees=&labels=requires+triage&template=bug_report.yaml

this is GREAT!!! we can PUT WORDS THAT THEY HAVE TO READ!!! W LINKS!!! HOORAY!!!

Reply:
I had also rigged by fork to test these.
>To test: to go my fork. Link https://github.com/CaselIT/sqlalchemy/issues/new/choose

it's quite neat